### PR TITLE
Landingpage

### PR DESCRIPTION
**Pull request by Rockyraven**
created langing page in react which is converted by HTML

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,18 @@
 import React from 'react'
+import { Route, Routes } from 'react-router-dom'
+import { Footer , Navbar} from './component'
```
Comment (arjun-computer-geek) [inline src/App.js:3]: This is good way of importing and exporting

File: src/component/footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import './footer.css'
+
+const Footer = () => {
+  return (
+    <footer class="footer">
+                
+    <div class="category-section">
```
Comment (arjun-computer-geek) [inline src/component/footer/Footer.jsx]: add className instead of class

File: src/component/footer/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            <li><a href="#" class="account-link">My Account</a></li>
+            <li><a href="#" class="account-link">My Order</a></li>
+            <li><a href="#" class="account-link">Wishlist</a></li>
+            <li><a href="#" class="account-link">Track Order</a></li>
+        </ul>
+    </div>
+    <div class="contact">
+        <h2 class="footer-heading">Contact Us</h2>
+        <ul>
+            <li><a href="#" class="contact-link">Location: Howarh, Kolkata West Bengal - 722101</a></li>
```
Comment (arjun-computer-geek) [inline src/component/footer/Footer.jsx]: use Link tag instead a tag

File: src/pages/landingpage/LandingPage.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import { Link } from 'react-router-dom'
+import { brandBanner, kidShoe, menShoe, womenShoe } from '../../assets'
+import './landingpage.css'
+
+export const LandingPage = () => {
+    return (
+        
+        <main>
+              <div class="brand-banner">
+                  <img src={brandBanner} alt="" class="banner"/>
```
Comment (arjun-computer-geek) [inline src/pages/landingpage/LandingPage.js]: add alt text

File: src/pages/landingpage/LandingPage.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { brandBanner, kidShoe, menShoe, womenShoe } from '../../assets'
```
Comment (arjun-computer-geek) [inline src/pages/landingpage/LandingPage.js:3]: use absolute path

**Review by arjun-computer-geek — APPROVED** [review]
Do suggested changes. Else looks good

File: src/component/footer/Footer.jsx (hunk 2 above)
Comment (Shubhamkr2610) [reply src/component/footer/Footer.jsx]: made the changes ... I think the same

**Review by Shubhamkr2610 — APPROVED** [review]
made the changes that @arjun-computer-geek  suggested ......every thing rest is good

**Pull request merged**